app/models/resource.py

```diff
@@ -89,7 +89,7 @@ def __repr__(self):
         return '<Resource \'%s\'>' % self.name
 
     @staticmethod
-    def generate_fake(count=20, center_lat=39.951021, center_long=-75.197243):
+    def generate_fake(count=10, center_lat=39.951021, center_long=-75.197243):
```

Comment:
This is minor but since this is an argument, you could also just pass in a number for count instead of modifying this method.